scripts/build/termux_step_massage.sh

```diff
@@ -79,6 +79,20 @@ termux_step_massage() {
 		termux_error_exit "Package contains hard links: $HARDLINKS"
 	fi
 
+	# Check so that package is not affected by https://github.com/android/ndk/issues/1614
+	SYMBOLS="$(readelf -s $($CC -print-libgcc-file-name) | grep "FUNC    GLOBAL HIDDEN" | awk '{print $8}')"
+	LIBRARIES="$(find lib -name "*.so")"
+	for lib in $LIBRARIES; do
+		for sym in $SYMBOLS; do
+			if ! readelf -h $lib &> /dev/null; then
+				continue
+			fi
+			if readelf -s $lib | grep "UND $sym" &> /dev/null; then
```

Comment:
The regexp for this line could be improved:

```bash
egrep 'NOTYPE[[:space:]]+GLOBAL[[:space:]]+DEFAULT[[:space:]]+UND[[:space:]]+'$sym'$'
```

where `NOTYPE` is specified in order to avoid matching "leaked builtins" from other shared libraries, though I suppose they have all been purged now.